plexpy/webauth.py

```diff
@@ -16,7 +16,8 @@
 #  along with Tautulli.  If not, see <http://www.gnu.org/licenses/>.
 
 
-# http://tools.cherrypy.org/wiki/AuthenticationAndAccessRestrictions
+# http://web.archive.org/web/20170210040849/http://tools.cherrypy.org/wiki/AuthenticationAndAccessRestrictions
```

Comment:
This is just a reference, there's no need to link to the wayback archive since the Github archive is sufficient.
```suggestion
```